src/palace/manager/service/google_drive/google_drive.py

```diff
@@ -116,7 +117,12 @@ def create_nested_folders_if_not_exist(
             folder = self.get_file(name=folder_name, parent_folder_id=parent_id)
 
             if not folder:
-                folder = self.service.files().create(body=body).execute()  # type: ignore[arg-type]
+                folder = (
+                    self.service.files()  # type: ignore[attr-defined]
+                    .create(body=body)
+                    .execute()
+                )
+            assert folder
```

Comment:
Minor: Did you mean to add this assertion? Might want to attach a message to it, or at least add a comment mentioning why the assertion is here.

Reply:
Yes: intentional and I add a message.